src/MultiSigSigner.sol

```diff
@@ -0,0 +1,179 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import {IDelegation} from "./interfaces/IDelegation.sol";
+import {ISigner} from "./interfaces/ISigner.sol";
+
```

Comment:
Note to self:

This is a singleton signer that enables many EOAs to maintain many multisig keys.